Cartfile.resolved

```diff
@@ -1,4 +1,4 @@
 binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon-ios.json" "10.0.0-beta.11"
 binary "https://api.mapbox.com/downloads/v2/carthage/mobile-maps-core/MapboxCoreMaps-dynamic.json" "10.0.0-beta.15"
-github "mapbox/mapbox-events-ios" "v0.10.7"
+github "mapbox/mapbox-events-ios" "v0.10.8"
 github "mapbox/turf-swift" "v2.0.0-alpha.2"
```

Comment:
Do we want to update Turf here and in the podspec as well?

Reply:
No, we still need this version for Carthage so that it gets built in library evolution mode. For CocoaPods, we will switch to 1.2.0 as part of #118.